setup.cfg

```diff
@@ -13,6 +13,9 @@ check_untyped_defs = True
 install_types = True
 non_interactive = False
 namespace_packages = True
+ignore_missing_imports = True
+implicit_optional = True
+ignore_errors = True
```

Comment:
is the ignoring of errors and missing imports a good idea? isn't that exactly what we want to find?